client/src/App.css

```diff
@@ -0,0 +1,16 @@
+* {
+  font-family: 'Montserrat', sans-serif;
+  color: rgb(72, 72, 72);
+  box-sizing: border-box;
+}
```

Comment:
You'll still want to avoid using * for anything but perhaps `border-box`. If you include it in your class components, they won't affect anyone else.